proto/elys/margin/query.proto

```diff
@@ -62,10 +63,20 @@ service Query {
   }
   
   // Queries a list of IsWhitelisted items.
-  rpc IsWhitelisted (IsWhitelistedRequest) returns (IsWhitelistedResponse) {
+  rpc GetIsWhitelisted (IsWhitelistedRequest) returns (IsWhitelistedResponse) {
     option (google.api.http).get = "/elys-network/elys/margin/is-whitelisted";
   
   }
+  
+  // Queries a list of Pool items.
+  rpc Pool    (QueryGetPoolRequest) returns (QueryGetPoolResponse) {
+    option (google.api.http).get = "/elys-network/elys/margin/pool/{index}";
+  
+  }
+  rpc GetPools (QueryAllPoolRequest) returns (QueryAllPoolResponse) {
```

Comment:
Could this be updated to Pools instead of `GetPools`

Reply:
@jelysn Okay. No problem.

Reply:
@jelysn Done, fixed.